### PR TITLE
Upgrade to Sylius 1.9

### PR DESCRIPTION
**Pull request by Tomanhez**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -13,19 +13,27 @@ jobs:
...
 
-        name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}, Node ${{ matrix.node }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
         strategy:
             fail-fast: false
             matrix:
                 php: [7.4, 7.3]
+                symfony: [^4.4, ^5.2]
+                sylius: [~1.8.0, ~1.9.0-RC.1]
```
Comment (AdamKasp) [inline .github/workflows/build.yml:23]:
```suggestion
                sylius: [~1.8.0, ~1.9.0]
```

Comment (lchrusciel) [inline .github/workflows/build.yml:23]:
```suggestion
                sylius: [~1.8.0, ~1.9.0]
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -13,19 +13,27 @@ jobs:
...
 
                 exclude:
                     -
                         php: 7.3
                         mysql: 8.0
+                    -
+                        sylius: ~1.8.0
+                        symfony: ^5.2
+                    -
+                        sylius: ~1.9.0-RC.1
```
Comment (AdamKasp) [inline .github/workflows/build.yml:35]:
```suggestion
                        sylius: ~1.9.0
```

Comment (lchrusciel) [inline .github/workflows/build.yml:35]:
```suggestion
                        sylius: ~1.9.0
```

File: .github/workflows/build.yml (hunk 1 above)
Comment (AdamKasp) [reply .github/workflows/build.yml:23]: we already published sylius 1.9 so IMO it should be without RC

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -13,19 +13,27 @@ jobs:
...
 
-        name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}, Node ${{ matrix.node }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
         strategy:
             fail-fast: false
             matrix:
                 php: [7.4, 7.3]
+                symfony: [^4.4, ^5.2]
+                sylius: [~1.8.0, ~1.9]
```
Comment (GSadee) [inline .github/workflows/build.yml:23]: I would be for:
```suggestion
                sylius: [~1.8.0, ~1.9.0]
```
to add next minor in clear way to our tests

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -13,19 +13,27 @@ jobs:
...
                 exclude:
                     -
                         php: 7.3
                         mysql: 8.0
+                    -
+                        sylius: ~1.8.0
+                        symfony: ^5.2
+                    -
+                        sylius: ~1.9
+                        php: 7.3
```
Comment (GSadee) [inline .github/workflows/build.yml:36]: Why did you add a conflict for Sylius 1.9 with PHP 7.3?

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -75,6 +83,23 @@ jobs:
                     restore-keys: |
                         ${{ runner.os }}-php-${{ matrix.php }}-composer-
 
+            -
+                name: Install Sylius version
+                run: |
+                    composer require "sylius/sylius:${{ matrix.sylius }}" --no-update
```
Comment (GSadee) [inline .github/workflows/build.yml:89]: I'm not sure if this is necessary as we have `Restrict Sylius version` in line 99, could you check that?

File: .github/workflows/build.yml (hunk 4 above)
Comment (Tomanhez) [reply .github/workflows/build.yml:36]: Was green, but we droped php 7.3 in Sylius 1.9

Comment (GSadee) [reply .github/workflows/build.yml]: Nope, we will drop PHP 7.3 in next minor version, we still support this version in 1.9: https://github.com/Sylius/Sylius/blob/1.9/composer.json#L22

**Comment by Tomanhez** [comment]
Closed for : https://github.com/Sylius/InvoicingPlugin/pull/212
